src/faq.md

```diff
@@ -190,55 +144,28 @@ appreciation through careful design choices and fair communication about them.
 
 ## Types
 
-### Q. Does Dart have type inference?
-
-Type inferencing is not something specified by the language specification, but
-it is something that implementations are free to do. It's important to remember
-that Dart has a dynamic type system, so type inferencing doesn't play the same
-role as it does in languages such as Haskell. However, IDEs might do some
-type inferencing, such as when you use var for local variables.
-
-### Q. Why are type annotations optional?
-
-We want to maintain the feel of a dynamically typed language, which is familiar
-to web developers. Mandatory types don't fit with that goal. Experience
-has also shown that full statically typed languages are sometimes too rigid,
-and we wanted Dart development to be more flexible for a wide range of
-developers.
+### Q. Is Dart a type-safe language?
 
-### Q. Why is the type system designed to be unsound?
+Dart is a type-safe language if you opt in to [strong mode].
```

Comment:
Technically, type safety is not actually the same thing as type soundness. And indeed, Dart is not actually type safe. And just to confuse things, "strongly typed", although it is frequently used, doesn't have a good definition, so "strong mode" can be a bit confusing.